package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -174,132 +167,319 @@ class WaterBridgeAnalysis_OtherFF(WaterBridgeAnalysis):
   w = MDAnalysis.analysis.hbonds.WaterBridgeAnalysis(u, 'resname ARG', 'resname ASP')
   w.run()
 
+The maximum number of bridging waters detected can be changed using the order keyword. ::
```

Comment:
This is now a long an expansive section. That's good. Eventually we want to massively restructure the docs #1175 and for this it would be useful if you could already structure the docs in such a way that we can just take a whole section and put it into a separate "how to use wbridge analysis".